src/main/java/uk/gov/pay/webhooks/deliveryqueue/managed/SendAttempter.java

```diff
@@ -0,0 +1,92 @@
+package uk.gov.pay.webhooks.deliveryqueue.managed;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import uk.gov.pay.webhooks.deliveryqueue.dao.WebhookDeliveryQueueDao;
+import uk.gov.pay.webhooks.deliveryqueue.dao.WebhookDeliveryQueueEntity;
+import uk.gov.pay.webhooks.message.WebhookMessageSender;
+
+import javax.inject.Inject;
+import javax.ws.rs.core.Response;
+import java.io.IOException;
+import java.net.http.HttpTimeoutException;
+import java.security.InvalidKeyException;
+import java.time.Duration;
+import java.time.InstantSource;
+import java.time.temporal.ChronoUnit;
+import java.util.Date;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class SendAttempter {
+    private static final Logger LOGGER = LoggerFactory.getLogger(SendAttempter.class);
+    private WebhookDeliveryQueueDao webhookDeliveryQueueDao;
+    private InstantSource instantSource;
+    private WebhookMessageSender webhookMessageSender;
+
+    @Inject
```

Comment:
I think this annotation can be removed if it will be initialised manually as in https://github.com/alphagov/pay-webhooks/pull/41/files#diff-bb525dc22ee461ff2cdcee4f8b1668836ea1dc55b47b754bf81cf36aaf76ae4fR43. 

I suspect it would be okay to let the DI manage this but I don't have an exact proposal here, happy to review after all the pieces are in